### ISSUE TITLE
long reads

### ISSUE THREAD

**Issue by davidvilanova** [marbl/Mash#116]
Hi,
Working with nanopore reads i´m using mash screen to identify the genomes from an input reads file as follow:

`mash  screen -w -p 3 refseq.genomes.k21s1000.msh reads_nanopore.fq.gz
`
So far it is doing pretty well as i recover a specific genome i was expecting in the metagenome (just using a small subset of the reads).

Is this the proper usage to get the mapped genomes of my nanopore reads ?

David

**Comment by tseemann** [comment]
@davidvilanova i don't think that is a plant genome, it looks like a plant pathogen; a microbe of some sort. it might have phage content similar to your bacterium

**Comment by davidvilanova** [comment]
Was not expecting this in a skin sample
